tests/test_stamps.py

```diff
@@ -81,6 +85,11 @@ def testIO(self):
         """
         self.roundtrip(make_stamps())
 
+    def testIOone(self):
+        """Test the class' write and readFits methods.
```

Comment:
Might be worth adding "when there is only a single stamp in the collection" to the docstring here